storage/StorageExample/Shared/ContentView.swift

```diff
@@ -208,12 +239,34 @@ struct ContentView: View {
         try await Auth.auth().signInAnonymously()
         authenticated = true
       } catch {
-        viewModel.errorFound = true
-        viewModel.errInfo = error
+        print("Not able to connect: \(error)")
+        Task { @MainActor in
+          viewModel.errorFound = true
+          viewModel.errInfo = error
+        }
         authenticated = false
       }
     }
   }
+
+  #if os(iOS)
+    func setImage(fromImage image: UIImage) -> Image {
+      return Image(uiImage: image)
+    }
+
+    func setImage(fromURL url: URL) -> Image {
+      return setImage(fromImage: UIImage(contentsOfFile: url.path)!)
+    }
+
+  #elseif os(macOS)
+    func setImage(fromImage image: NSImage) -> Image {
+      return Image(nsImage: image)
+    }
+
+    func setImage(fromURL url: URL) -> Image {
+      return setImage(fromImage: NSImage(contentsOfFile: url.path)!)
+    }
+  #endif
```

Comment:
optional: Does this work? If so, it avoids the need for duplicate `setImage(fromURL url: URL) -> Image` APIs.

```suggestion
  #if os(iOS)
    func setImage(fromImage image: UIImage) -> Image {
      return Image(uiImage: image)
    }
  #elseif os(macOS)
    func setImage(fromImage image: NSImage) -> Image {
      return Image(nsImage: image)
    }
  #endif

  func setImage(fromURL url: URL) -> Image {
    return setImage(fromImage: .init(contentsOfFile: url.path)!)
  }

```

Reply:
That's neat! It is working :)

Reply:
oh, cool!